.github/workflows/test.yml

```diff
@@ -49,11 +49,15 @@ jobs:
     - name: Install requirements (Python 3)
       if: ${{ matrix.ckan-version != '2.7' && matrix.ckan-version != '2.8' && matrix.ckan-version != '2.9-py2'}}
       run: |
+        pip install cython
+        pip install cchardet
```

Comment:
I guess these are not needed here if they are in requirements.txt

Reply:
I thought so, but it's needed there, otherwise the github workflow tried to install first cchardet before cython. I tried to put them first in the requirements.txt but this was the only way to fix it

Reply:
Look at this, with the cython and cchardet only in requirements:

https://github.com/frictionlessdata/ckanext-datapackager/actions/runs/2759356061